crates/ruff_linter/src/rules/pyflakes/snapshots/ruff_linter__rules__pyflakes__tests__preview__F401_F401_27__all_mistyped____init__.py.snap

```diff
@@ -0,0 +1,26 @@
+---
+source: crates/ruff_linter/src/rules/pyflakes/mod.rs
+---
+__init__.py:5:15: F401 [*] `.unused` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
+  |
+5 | from . import unused # F401: change to redundant alias b/c __all__ cannot be located
+  |               ^^^^^^ F401
+  |
+  = help: Use a redundant alias: `.unused`
```

Comment:
This message feels kinda confusing to me right now; would it be possible for the message to say something like this instead?

```suggestion
  = help: Use an explicit re-export: `from . import unused as unused`
```